src/redux/actionTypes/experimentSettings.js

```diff
@@ -0,0 +1,25 @@
+const EXPERIMENT_SETTINGS = 'experimentSettings';
```

Comment:
just to make sure I got it right, the design of these actions is with the thought to use them for any settings that will influence the data in the experiment, for instance we will use them for the data integration module, for the QC filters too. Is that correct?

Reply:
I also envisioned this store to be used for more general experiment settings/metadata in the future, such as the name of the private investigator, team, organism data, last modified, name of experiment, etc.

Reply:
great, this will be very needed in the future